test/manipulation.test.js

```diff
@@ -1920,10 +2039,15 @@ describe('manipulation', function() {
 
     it('should not update instances that do not satisfy the where condition',
         function(done) {
-          Person.update({name: 'Harry Hoe'}, {name: 'Marta Moe'}, function(err,
+          var filter = isCassandraConnector ? {id: idHarry} : {name: 'Harry Hoe'};
```

Comment:
used connectorCapabilities.deleteWithOtherThanId and connectorCapabilities.reportDeletedCount